src/content/topics/sdk/features/flags-from-files.mdx

```diff
@@ -246,37 +246,7 @@ To configure the client:
 ```
 
 </CodeTabItem>
-</CodeTabs>
-
-### Erlang
-
-To configure the client:
-
-<CodeTabs
-  defaultValue="erlang"
-  values={[
-    { label: 'Erlang', value: 'erlang', },
-  ]
-}>
-<CodeTabItem value="erlang">
-
-```erlang
-  ldclient:start_instance("sdk key", #{
-    file_datasource => true,
-    send_events => false,
-    file_paths => ["file1.json", "file2.yaml"],
-    feature_store => ldclient_storage_map,
-    file_auto_update => true,
-    file_poll_interval => 1000
-  })
-
-  %% In the Erlang SDK automatic reloading uses a polling mechanism.
-  %% The default interval is 1000ms, but you can control it with
-  %% the file_poll_interval configuration.
-```
-
-</CodeTabItem>
-</CodeTabs>
+</CodeTabs> 
```

Comment:
```suggestion
### Erlang

To configure the client:

<CodeTabs
  defaultValue="erlang"
  values={[
    { label: 'Erlang', value: 'erlang', },
  ]
}>
<CodeTabItem value="erlang">

```erlang
  ldclient:start_instance("sdk key", #{
    file_datasource => true,
    send_events => false,
    file_paths => ["file1.json", "file2.yaml"],
    feature_store => ldclient_storage_map,
    file_auto_update => true,
    file_poll_interval => 1000
  })

  %% In the Erlang SDK automatic reloading uses a polling mechanism.
  %% The default interval is 1000ms, but you can control it with
  %% the file_poll_interval configuration.
```

</CodeTabItem>
</CodeTabs> 
```